ADR/0018-apps-continuous-perf-testing.md

```diff
@@ -78,5 +78,6 @@ Cons:
 - Horreum is used by multiple teams without any capacity issues, but Horreum itself was not perf&scale tested formally, so there might be some scaling issues.
 - We would need to develop StoneSoup UI to get graphs/results from Horreum and to allow users to configure the change detection parameters for their tests.
 - If we need one Horreum instance per workspace, that would require further development work (i.e. operator for provisioning per worksapce, data backup/restore etc).
+- As the performance testing pipelines can be some heavy on resources, attribution of costs to users/workspaces might be tricky. Maybe we could add Horreum to scope of <https://issues.redhat.com/browse/STONE-215>.
```

Comment:
There are 2 components to attributable cost here I think.  The resources/cost to actually run the tests which could be substantial for a period of time,  and the resource cost of Horreum processing a result.  
- 1: resource cost of running tests: I would have thought this could be accounted for using the same methodology for the build pipelines as it would be part of the pipeline
- 2: resource cost for Horreum: will be dependent on size and frequency of data ingest and processing. This might be tricky to attribute to individual users/piplines, however storage resource usage of historic data would be easier to attribute to users/workspaces

Reply:
I agree that 1 would likely fall under "for free"

Reply:
Right. Exploded these to two distinct lines: cfc1704